### PR TITLE
Abass/review contest

### PR DESCRIPTION
**Pull request by abassawo**
Con 39 - Review contest before submitting.

### CODE REVIEW

**Comment by jmccormack200** [comment]
restart jenkins build

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -3,9 +3,9 @@
           package="io.intrepid.contest"
     >
 
-    <uses-permission android:name="android.permission.INTERNET"/>
```
Comment (gsintrepid) [inline app/src/main/AndroidManifest.xml]: I think you deleted this by mistake (don't delete it please! haha)

File: app/src/main/java/io/intrepid/contest/screens/contestcreation/namecontest/NameContestFragment.java [hunk 2]
```diff
@@ -59,9 +59,12 @@ public final void onTextChanged(CharSequence newName) {
 
     @Override
     public void setNextEnabled(boolean enabled) {
+        if (enabled) {
+            trophyIcon.setVisibility(GONE);
+        } else {
+            trophyIcon.setVisibility(VISIBLE);
+        }
         contestEditorActivity.setNextEnabled(enabled);
-        int trophyVisibility = enabled ? GONE : VISIBLE;
```
Comment (gsintrepid) [inline app/src/main/java/io/intrepid/contest/screens/contestcreation/namecontest/NameContestFragment.java]: I thought you were going to keep this ternary?

File: app/src/main/res/layout/fragment_review_contest.xml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            style="@style/AppTheme.TextViewLabel"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_margin="0dp"
+            android:text="@string/categories"
+            android:textColor="@color/colorLightGray"
+            tools:visibility="visible"
+            />
+
+        <include layout="@layout/include_recycler_view"/>
```
Comment (gsintrepid) [inline app/src/main/res/layout/fragment_review_contest.xml]: Actually can you change this layout title for something specific to the review screen?

File: app/src/main/res/layout/fragment_review_contest.xml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+    <TextView
+        android:id="@+id/contest_title_button"
+        style="@style/AppTheme.TextViewLabel"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:clickable="true"
+        />
+
+    <View
```
Comment (gsintrepid) [inline app/src/main/res/layout/fragment_review_contest.xml]: These properties can be a style!
So you could use it as `<View style="@style/AppTheme.HorizontalRow" />`

File: app/src/main/res/layout/fragment_review_contest.xml [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+    <TextView
+        android:id="@+id/contest_description_button"
+        style="@style/AppTheme.TextViewLabel"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:clickable="true"
+        />
+
+    <View
```
Comment (gsintrepid) [inline app/src/main/res/layout/fragment_review_contest.xml]: [Same using the style]

File: app/src/main/res/layout/fragment_review_contest.xml [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+        android:layout_height="1dp"
+        android:background="@color/colorLightGray"
+        />
+
+    <TextView
+        android:id="@+id/contest_categories_hint"
+        style="@style/AppTheme.TextViewLabel"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_margin="0dp"
```
Comment (gsintrepid) [inline app/src/main/res/layout/fragment_review_contest.xml]: You probably don't need this margin

File: app/src/test/java/io/intrepid/contest/screens/contestcreation/NewContestPresenterTest.java [hunk 7]
```diff
@@ -67,7 +67,7 @@ public void onBackButtonClickedShouldNavigateBackwards() {
     }
 
     @Test
-    public void onBackTwo() {
+    public void onBackTwo(){
```
Comment (gsintrepid) [inline app/src/test/java/io/intrepid/contest/screens/contestcreation/NewContestPresenterTest.java]: I think in some branch you had removed this test -- it's the same as the previous test

**Comment by abassawo** [comment]
restart jenkins build

**Pull request closed without merging**